api/db.mjs

```diff
@@ -7,11 +7,18 @@ const insertOne = async (insertData) => {
   return response.insertedId;
 };
 
+const insertReview = async (insertData) => {
+  const db = await getDB();
+  const testCollection = db.collection("reviews");
+  const response = await testCollection.insertOne(insertData);
```

Comment:
There is a little bit of a security and data structure concern here. Since we are inserting all of the data that we get from the API request, then technically, someone could flood the DB with anything they want even thought it does not match the expected data. This could lead to data we can't use as expected.

For example, since we plan to store reviews, someone might pass in an object with a capital R for review and someone else might pass in an object with a lowercase r:

```json
{"Review": "some review"}
```

```json
{"review": "some review"}
```

These will both be valid inputs an will be saved to the database, but they should not be valid because it will make pulling the data harder in the future as we will have to account for data in different formats

Instead, lets give the data a structure so that the data is always the same for all of the fields we expect, for example:

```javascript
const response = await testCollection.insertOne({review: insertData.review, someOtherField: insertData.someOtherField});
```